zeppelin-web/src/app/home/home.html

```diff
@@ -32,13 +33,24 @@
       </i>
     </a>
     <a style="text-decoration: none;">
-      <i style="font-size: 13px; margin-left: 2px; cursor: pointer; text-decoration: none;"
+      <!-- if note is not in trash -->
+      <i ng-if="!node.isTrash" style="font-size: 13px; margin-left: 2px; cursor: pointer; text-decoration: none;"
```

Comment:
How about creating a `class` for this `style` into "home.css" file?
Same `style` properties are being used in this file several times.